docs/setup-guide/arktos-with-mizar-cni.md

```diff
@@ -0,0 +1,40 @@
+# Arktos to Enforce the Multi-tenant with Mizar Network Feature
+
+This document captures the steps applied to an Arktos cluster lab enabling the unique multi-tenant network feature. The machines in this lab used are AWS EC2 t2-large (2 CPUs, 8GB mem), Ubuntu 18.04 LTS.
+
+The steps might change with the progress of development.
+  
+1. Update Kernel
+To check kernel, run following command
+
+```bash
+uname -a
+```
+
+To update Kernel, download and run:
+If it is `5.6.0-rc2` then you can skip downloading ```kernelupdate.sh```
+```bash
+wget https://raw.githubusercontent.com/CentaurusInfra/mizar/dev-next/kernelupdate.sh
+bash kernelupdate.sh
+```
+
+2. Start Arktos cluster
+```bash
+CNIPLUGIN=mizar ./hack/arktos-up.sh
+```
```

Comment:
I tried this after running setup-dev-node.sh and hit the following issue:
```
ubuntu@ip-192-168-1-15:~/arktos$ CNIPLUGIN=mizar ./hack/arktos-up.sh
WARNING : The kubelet is configured to not fail even if swap is enabled; production deployments should disable swap.
WARNING : This script MAY be run as root for docker socket / iptables functionality; if failures occur, retry as root.
cni plugin is mizar; arktos will use mizar to provision pod network
./hack/../hack/arktos-cni.rc: line 72: pip3: command not found
--2021-09-12 17:39:27--  https://raw.githubusercontent.com/CentaurusInfra/mizar/dev-next/bootstrap.sh
Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 185.199.108.133, 185.199.109.133, 185.199.110.133, ...
Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|185.199.108.133|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 2969 (2.9K) [text/plain]
Saving to: ‘/tmp/bootstrap.sh’

bootstrap.sh                      100%[============================================================>]   2.90K  --.-KB/s    in 0s      

2021-09-12 17:39:27 (25.5 MB/s) - ‘/tmp/bootstrap.sh’ saved [2969/2969]

Hit:1 http://us-west-1.ec2.archive.ubuntu.com/ubuntu bionic InRelease
Hit:2 http://us-west-1.ec2.archive.ubuntu.com/ubuntu bionic-updates InRelease
Hit:3 http://us-west-1.ec2.archive.ubuntu.com/ubuntu bionic-backports InRelease
Hit:4 http://security.ubuntu.com/ubuntu bionic-security InRelease               
Reading package lists... Done                      
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Note, selecting 'python-scapy' instead of 'scapy'
Package docker-ce is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Package 'docker-ce' has no installation candidate
update-alternatives: using /usr/bin/python3.6 to provide /usr/bin/python3 (python3) in auto mode
update-alternatives: error: alternative path /usr/bin/python3.7 doesn't exist
update-alternatives: error: alternative /usr/bin/python3.7 for python3 not registered; not setting
sudo: pip3: command not found
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   624  100   624    0     0   2971      0 --:--:-- --:--:-- --:--:--  2971
100 6660k  100 6660k    0     0  17.1M      0 --:--:-- --:--:-- --:--:-- 17.1M
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 44.7M  100 44.7M    0     0  77.2M      0 --:--:-- --:--:-- --:--:-- 77.1M
/tmp/bootstrap.sh: line 62: logout: not login shell: use `exit'
ubuntu@ip-192-168-1-15:~/arktos$
```

./hack/arktos-up.sh works just fine. 

It is extremely disappointing that you have not even bothered to test your own work..

Reply:
Made changes in the bootstrap script to fix the issue, Thanks.

Reply:
Why do we need to run bootstrap script? What does the bootstrap script do that node-init does not do?

Reply:
We have added the required steps and removed the bootstrap dependency